src/frontend.c

```diff
@@ -2910,16 +2910,28 @@ int frontend(struct program *prg, int nfile, char **fname, dictmap_callback_t di
 					}
 				} else if(1 == sscanf(str, "font-style : %s", param)) {
 					if(!strcmp(param, "italic") || !strcmp(param, "oblique")) {
-						bc->style = STYLE_ITALIC;
+						bc->style |= STYLE_ITALIC;
+					}else if(!strcmp(param, "normal")) {
+						bc->unstyle |= STYLE_ITALIC;
 					}
 				} else if(1 == sscanf(str, "font-weight : %s", param)) {
 					if(!strcmp(param, "bold")) {
-						bc->style = STYLE_BOLD;
+						bc->style |= STYLE_BOLD;
+					}else if(!strcmp(param, "normal")) {
+						bc->unstyle |= STYLE_BOLD;
 					}
 				} else if(1 == sscanf(str, "font-family : %s", param)) {
-					// %s stops at first whitespace, so use str:
+					// %s stops at first whitespace, so use str instead
 					if(strstr(str, "monospace")) {
-						bc->style = STYLE_FIXED;
+						bc->style |= STYLE_FIXED;
+					}else if(strcmp(param, "inherit")) { // Something that's not monospace was specified, and it was *not* inherit
+						bc->unstyle |= STYLE_FIXED;
+					}
+				} else if(1 == sscanf(str, "font-decoration : %s", param)) {
+					if(strstr(str, "reverse")) { // This is not a standard CSS property, but there is no standard CSS property for reverse video, and unrecognized property values are explicitly not an error in CSS
```

Comment:
Mild preference for `reverse-video`, if that works for you? (Less ambiguous and less likely to conflict with (the admittedly unlikely possibility of) some future version of CSS defining this differently.